VEDA_OS01/management/commands/create_oauth_app_client.py

```diff
@@ -0,0 +1,146 @@
+"""
+Management command used to create an OAuth client in the database.
+"""
+
+from django.contrib.auth import get_user_model
+from django.core.exceptions import ValidationError
+from django.core.management.base import BaseCommand, CommandError
+from django.core.validators import URLValidator
+from oauth2_provider.models import Application
+
+CLIENT_TYPES = [client_type[0] for client_type in Application.CLIENT_TYPES]
+GRANT_TYPES = [client_type[0] for client_type in Application.GRANT_TYPES]
+
+
+class Command(BaseCommand):
+    """
+    create_oauth_app_client command class
+    """
+    help = 'Create a new OAuth Application Client. Outputs a serialized representation of the newly-created Client.'
+
+    def add_arguments(self, parser):
```

Comment:
@muhammad-ammar can we add default values for these arguments, idea is to just run it for sandbox.

Reply:
I will prefer to remain this as it is.